File: public/phase-2-20160620022203/PYTHON/pocket-money.py
#!/usr/bin/python
# Name : Pocket money
# input:
# n : the number of exams
# r :  the maximum score
# avg : average score required to double his pocket money
# ab : ab corresponds to ai bi
elements = raw_input().strip().split(' ')
n = int(elements[0])
r = int(elements[1])
avg = int(elements[2])
ab = []
for _ in xrange(n):
	ab_elements = raw_input().strip().split(' ')
	ab.append(map(int, ab_elements))


# write your code here
# store your results in `result`

# output

# Dummy Data
result = 4

print(result)

File: public/phase-2-20160620022203/C/pocket-money.c
// Name : Pocket money
#include <stdio.h>
int main() {
	// input:
	// n : the number of exams
	// r :  the maximum score
	// avg : average score required to double his pocket money
	// ab : ab corresponds to ai bi
	int n, r, avg, ab[10000][2], result=4, idx, jdx;
	char ab_tmp[2];
	scanf("%d %d %d", &n, &r, &avg);
	for(idx = 0; idx< n; idx++) {
		scanf("%s", &ab_tmp[0]);
		for(jdx = 0;jdx<2;jdx++) {
			ab[idx][jdx] = ab_tmp[jdx];
		}
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%d", result);
	return 0;
}